client/pages/org-edit-modal/modal.template.html.slim

```diff
@@ -4,19 +4,13 @@ form#org-edit-modal novalidate=true name='orgForm' ng-submit='orgForm.$valid &&
     h3.modal-title ng-if='editedOrg.id' Edit {{ organization | orgName }}
     h3.modal-title ng-if='!editedOrg.id' New Organization
   .modal-body
-    .form-group ng-class='{"has-error": orgForm.name.$dirty && orgForm.name.$invalid}'
-      label for='orgName' Name
-      input#orgName.form-control type='string' name='name' placeholder='Name (used in URLs)' required=true maxlength=50 ng-pattern='/^[a-z0-9]+(?:-[a-z0-9]+)*$/i' unique-org-name=true autofocus=true ng-model='editedOrg.name' ng-model-options='{updateOn: "default blur", debounce: {default: 500, blur: 0}}'
+    .form-group ng-class='{"has-error": orgForm.displayName.$dirty && orgForm.displayName.$invalid}'
+      label for='orgDisplayName' Name
+      input#orgDisplayName.form-control type='string' name='displayName' maxlength=50 ng-model='editedOrg.displayName' unique-org-name=true
       p.help-block
-        | This is the identifier of the organization and is used in URLs.
-        |  It can only contain alphanumeric characters and single hyphens (not at the beginning or end).
-      .alert.alert-danger ng-if='orgForm.name.$error.uniqueOrgName'
+        | This is the name of the organization as shown in Probe Dock.
```

Comment:
This message no longer makes sense. The "as shown in Probe Dock" was meant to differentiate it from the identifier. I would replace the full message with "The unique name of the organization, up to 50 characters."